rust/gui-client/src-tauri/src/client/gui.rs

```diff
@@ -226,6 +216,11 @@ pub(crate) fn run(
                         tracing::warn!(
                             "Will crash / error / panic on purpose in {delay} seconds to test error handling."
                         );
+                        telemetry::add_breadcrumb(telemetry::Breadcrumb {
+                            ty: "fail_on_purpose".into(),
+                            message: Some("Will crash / error / panic on purpose to test error handling.".into()),
+                            ..Default::default()
+                        });
```

Comment:
I guess we are not leaving this in?

Reply:
I'd like to, it's handy for testing error handling. The flags for it were already present since they were left in after testing the existing crash handling and logging.